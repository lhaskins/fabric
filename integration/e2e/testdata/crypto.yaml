
OrdererOrgs:
  - Name: ExampleCom
    Domain: example.com
    Specs: 
      - Hostname: orderer0


PeerOrgs: 
  - Name: Org1ExampleCom
    Domain: org1.example.com
    EnableNodeOUs: false
    Template:
      Count: 1
    Users:
      Count: 1
			
  - Name: Org2ExampleCom
    Domain: org2.example.com
    EnableNodeOUs: false
    Template:
      Count: 1
    Users:
      Count: 1
			