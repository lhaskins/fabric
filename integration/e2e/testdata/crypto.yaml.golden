
OrdererOrgs:
  - Name: ExampleCom
    Domain: example.com
    CA:
        Country: US
        Province: California
        Locality: San Francisco
    Specs: 
      - Hostname: orderer0


PeerOrgs: 
  - Name: Org1ExampleCom
    Domain: org1.example.com
    EnableNodeOUs: true
    CA:
        Country: US
        Province: California
        Locality: San Francisco
    Template:
      Count: 2
    Users:
      Count: 2

  - Name: Org2ExampleCom
    Domain: org2.example.com
    EnableNodeOUs: true
    CA:
        Country: US
        Province: California
        Locality: San Francisco
    Template:
      Count: 2
    Users:
      Count: 2
